### PR TITLE
Adding cluster_short_name in alert message instead of integration_id

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/monitoring-integration#403

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/brick_utilization_handler.py [hunk 1]
```diff
@@ -19,9 +19,8 @@ class BrickHandler(AlertHandler):
 
     def __init__(self):
         AlertHandler.__init__(self)
-        self.template = "tendrl.clusters.{integration_id}.nodes."\
-            "{host_name}.bricks.{brick_path}.utilization."\
-            "percent-percent_bytes"
+        self.template = "tendrl[.]name[.]{integration_id}[.]nodes[.]"\
+            "{host_name}[.]bricks[.]{brick_path}[.]"
```
Comment (GowthamShanmugam) [inline tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/brick_utilization_handler.py]: In old regular expression template, we always have to give a word at the end. For example in this template "percent-percent_bytes" is given. But this template will do exact match only, no need any extra word at the end.

So whenever metric changes after brick_path we don't need to change the template.

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @cloudbehl please review

**Comment by shtripat** [comment]
@cloudbehl please check my comments regarding hierarchy in graphite. Is it for writing to graphite or referring data from graphite symlink? Even if its symlink it should be like `tendrl/names/{cluster_short_name}/volumes/{volume_name}`. please check and comment accordingly.

File: tendrl/monitoring_integration/alert/utils.py [hunk 2]
```diff
@@ -214,3 +214,20 @@ def find_volume_name(integration_id, hostname, brick_path):
             }
         )
         raise ex
+
+
+def find_cluster_short_name(integration_id):
+    try:
+        cluster_name = etcd_utils.read(
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/utils.py]: i missed this :)

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py [hunk 3]
```diff
@@ -19,8 +19,8 @@ class VolumeHandler(AlertHandler):
 
     def __init__(self):
         AlertHandler.__init__(self)
-        self.template = "tendrl.clusters.{integration_id}.volumes."\
-            "{volume_name}.pcnt_used"
+        self.template = "tendrl[.]name[.]{integration_id}[.]volumes[.]"\
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py]: i will change this @shtripat

**Comment by GowthamShanmugam** [comment]
@shtripat Changes done

File: tendrl/monitoring_integration/alert/utils.py [hunk 4]
```diff
@@ -214,3 +214,20 @@ def find_volume_name(integration_id, hostname, brick_path):
...
         )
         raise ex
+
+
+def find_cluster_short_name(integration_id):
+    try:
+        cluster = NS.tendrl.objects.Cluster(
+            integration_id=integration_id
+        ).load()
+        return cluster.short_name
```
Comment (r0h4n) [inline tendrl/monitoring_integration/alert/utils.py]: shouldnt you `return cluster.short_name or cluster.integration_id`  

@GowthamShanmugam @nthomas-redhat

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py [hunk 5]
```diff
@@ -63,7 +63,7 @@ def format_alert(self, alert_json):
                     "Volume utilization on %s in "
                     "%s back to normal" % (
                         alert['tags']['volume_name'],
-                        alert['tags']['integration_id']
+                        alert['tags']['cluster_short_name']
```
Comment (r0h4n) [inline tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py]: why are you assuming "cluster_short_name" will always be set? 

what happens at this line if there is no "cluster_short_name"

File: tendrl/monitoring_integration/alert/utils.py (hunk 4 above)
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/utils.py]: I thought if the username is not given by user then backed is assigning integration_id as  short_name in cluster object. I will change this

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py (hunk 5 above)
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py]: If cluster_short_name is not present then I will assign  integration_id in alert['tags']['cluster_short_name']

**Pull request merged**